experimental/QuadFormAndIsom/src/embeddings.jl

```diff
@@ -1237,10 +1348,15 @@ $L'$ isometric to $M$, and $N'$ is the orthogonal complement of $M'$ in $L'$.
 If `T == false`, then $V$ will always be the empty list. If `T == true`, then
 the content of $V$ depends on the value of the symbol `classification`. There
 are 4 possibilities:
```

Comment:
```suggestion
are four possibilities:
```

Reply:
Thanks! I have spread the suggestion to the other docstrings